game.py

```diff
@@ -63,23 +34,39 @@ def get_letter_from_user(snowman_word_dict, wrong_guesses_list):
             valid_input = True
 
     return user_input_string
-    
+
+def get_word_progress(snowman_word, snowman_word_dict):
+    for letter in snowman_word:
+        if not snowman_word_dict[letter]:
+            return False
+    return True
```

Comment:
This function was already defined later in this file. Leaving it where it was was fine.